src/helpers/geocode.js

```diff
@@ -68,7 +68,7 @@ export const searchBoundaryLayer = async (searchStr, boundaryId) => {
   const searchParams = {
     srs: 4326,
     search: searchStr,
-    v: 1, // cache bust 2022-05-09
```

Comment:
An optional update is to create a new `const v = 2` at the top of this file and then use it for three other places it appears.

Reply:
> An optional update is to create a new `const v = 2` at the top of this file and then use it for three other places it appears.

True, that would be more efficient wouldn't it. I'll make the changes...